CIScripts/Test/PesterLogger/PesterLogger.Tests.ps1

```diff
@@ -0,0 +1,58 @@
+﻿$here = Split-Path -Parent $MyInvocation.MyCommand.Path
+$sut = (Split-Path -Leaf $MyInvocation.MyCommand.Path) -replace '\.Tests\.', '.'
+. "$here\$sut"
+
+Describe "PesterLogger" {
```

Comment:
Can we have a test case that actually invokes Add-ContentForce?
It may also be useful to verify that Push-Location and Pop-Location do not interfere with scope handling/file path.